src/AdvApi32.Desktop/AdvApi32+TOKEN_INFORMATION_CLASS.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="TOKEN_INFORMATION_CLASS"/> nested type.
+    /// </content>
+    public partial class AdvApi32
+    {
+        /// <summary>
+        /// Specify the type of information being assigned to or retrieved from an access token.
+        /// </summary>
+        public enum TOKEN_INFORMATION_CLASS
+        {
+            TokenUser = 1,
+            TokenGroups,
```

Comment:
For once it's actually defined as a real enum in native code : https://msdn.microsoft.com/en-us/library/windows/desktop/aa379626(v=vs.85).aspx with the same value generation